src/hooks/token.js

```diff
@@ -2,15 +2,6 @@ import WFRP_Utility from "../system/utility-wfrp4e.js";
 import passengerRender from "../system/passengerRender.js"
 
 export default function() {
-  // Adds tooltips to conditions in the condition menu
-  Hooks.on("renderTokenHUD", async (obj, html) => {
-    for (let condition of html.find("img.effect-control")) {
-      condition.title = game.wfrp4e.config.conditions[condition.dataset["statusId"]]
-      if (condition.dataset["statusId"] == "dead")
-        condition.title = "Dead"
-    }
-  })
-
```

Comment:
This I removed because it would double the tooltips. 
![image](https://github.com/user-attachments/assets/176cab4c-7f6a-49d9-80e3-4cdac84cf4fd)